src/components/InventoryV2/index.tsx

```diff
@@ -226,12 +238,12 @@ export default function Inventory(props: InventoryProps): JSX.Element {
               {
                 id: 'batches_by_species',
                 label: strings.BY_SPECIES,
-                children: <InventoryListBySpecies />,
+                children: <InventoryListBySpecies setReportData={setReportData} />,
               },
               {
                 id: 'batches_by_nursery',
                 label: strings.BY_NURSERY,
-                children: <InventoryListByNursery />,
+                children: <InventoryListByNursery setReportData={setReportData} />,
```

Comment:
interesting, this doesn't error out? setReportData type is different here and in the component props

Reply:
It doesn't error out, I see we do that in other places. Should I change it to be React.Dispatch ... ?

Reply:
My suggestion is to not make it a React.Dispatch, that's a very internal semantic to this component. Leave it as is maybe if we are doing similar stuff elsewhere.